app/code/Magento/MediaContentSynchronizationCms/etc/di.xml

```diff
@@ -14,6 +14,15 @@
             </argument>
         </arguments>
     </type>
+    <type name="Magento\MediaContentSynchronizationApi\Model\SynchronizerIdentitiesPool">
+        <arguments>
+            <argument name="synchronizers" xsi:type="array">
+                <item name="media_content_cms_synchronizeIdentity"
```

Comment:
Please use either `underscore_way` or `camelCaseWay` of naming, but not the mix. Also this is the item inside `synchronizers` property, so I think the `synchronizeIdentity` suffix can be dropped from the naming
```suggestion
                <item name="media_content_cms"
```